packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -215,7 +215,7 @@ const SettingsSidebarContainer = ({
   return (
     <nav
       className={classNames(
-        "no-scrollbar bg-muted fixed bottom-0 left-0 top-0 z-20 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:flex",
+        "no-scrollbar bg-muted fixed bottom-0 left-0 top-0 z-20 flex max-h-screen w-56 flex-col space-y-1 overflow-x-hidden overflow-y-scroll px-2 pb-3 transition-transform max-lg:z-10 lg:sticky lg:z-0 lg:flex",
```

Comment:
@puru-khedre Although this works, shouldn't the top banner's z index be changed instead?

Reply:
I tried to fix it by changing the value of z-index but it didn't work.

Reply:
@puru-khedre - TopBanner component has `static` position, and z-index property doesn't work on `static` elements. Try changing it to `relative` and then add `z-index` property to see the changes.

Reply:
@puru-khedre wanna try the above?